plasma_framework/contracts/src/exits/payment/spendingConditions/PaymentOutputToPaymentTxCondition.sol

```diff
@@ -20,15 +20,16 @@ contract PaymentOutputToPaymentTxCondition is IPaymentSpendingCondition {
     /**
      * @notice Checks if given output has been spent by owner in given spending transaction.
      * @param _outputGuard bytes that hold the address of owner directly.
-     * @param _utxoPos serves as the identifier of output.
+     * @dev _utxoPos not used, serves as the position identifier of output.
+     * @param _outputId serves as the identifier of output (input pointer).
      * @param _spendingTx The rlp encoded transaction that spends the output.
      * @param _inputIndex The input index of the spending transaction that points to the output.
      * @param _signature The signature of the output owner.
      */
     function verify(
         bytes32 _outputGuard,
-        uint256 _utxoPos,
-        bytes32 /*_outputId*/,
+        uint256, /*_utxoPos  NOTE: It's unclear how & if this will be used at all, see: https://github.com/omisego/plasma-contracts/pull/212*/
+        bytes32 _outputId,
```

Comment:
This naming would conflict with our normal definition of `outputId` as currently `outputId` means a specific way of computing the id. In here it means a more generic identifier (both utxoPos and outputId) and would lead to confusion.

IDK, how about sth like `outputIdentifier` ? (I know this...is not really better) or hope you can come up with some better naming :p.

Reply:
This is pointing to input of the tx in-question. Maybe inputPointer or inputIdentifier ?